### DISCUSSION TITLE
Welcome to oracle-sql-plsql-b3 Discussions!

### DISCUSSION THREAD

**Discussion by acrlakshman** [exaslate-courses/oracle-sql-plsql-b3#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Tharun1596** [comment]
Hello Everyone 
This is Tharun  kumar chancharapu 
I have completed my master's from southeast Missouri State University in computer science.
I'm new to programming field.
I'm here to learn oracle SQL and become a data engineer,
Thank you.

**Reply by acrlakshman** [reply]
Thanks @Tharun1596 for the introduction.

**Comment by YaminiYadavv** [comment]
Hello Everyone
My name is Yamini Danaboina
I am pursuing my master's in information management in St. Francis College
I am delighted to be a part of this class and learn PL/SQL query language; I am a beginner in coding language Journey.
I would like to welcome all with warm regards.
Thank You.

**Reply by acrlakshman** [reply]
Thank you @YaminiYadavv  for introducing yourself.
